### ISSUE TITLE
Split timestamp in filename so it is more approachable

### ISSUE THREAD

**Issue by paulcsmith** [luckyframework/lucky_migrator#108]
Right now it bunches everything together: `20180612204856_my_migration`

It'd be a lot nicer like this: `2018_06_12_20_48_56_my_migration` or something along those lines

**Comment by mikeeus** [comment]
I agree the long number is confusing but I think your other alternative has too many underscores. What about only splitting the date and time like 20180612_204856_my_migration?

**Comment by paulcsmith** [comment]
I think you’re right that I added too many. Maybe just around the year month and day would work too!

**Comment by mikeeus** [comment]
yeah I see, something like this 2018_06_12_204856_my_migration?

**Comment by paulcsmith** [comment]
Yeah, I think that is a perfect balance!
